Initium-Odp/src/com/universeprojects/miniup/server/HtmlComponents.java

```diff
@@ -241,6 +241,7 @@ public static String generateTradeInvItemHtml(CachedEntity item, List<CachedEnti
 		String result = "";
 		   result+="<div class='invItem' ref="+item.getKey().getId()+">";
 		   result+="<div class='main-item'>";
+		   result+="<input type=checkbox>";

```

Comment:
I'm sure it'll be ok but proper html formatting is something more like this: <input type='checkbox'/>